types/KeyguardRequests.backup

```diff
@@ -71,7 +71,7 @@ type SignMessageResult = {
     signature: Uint8Array
```

Comment:
Since you renamed this file to `.backup`, is it still necessary or can it be removed?

Reply:
removed!